osu.Game/Screens/Play/Break/BreakInfoLine.cs

```diff
@@ -23,69 +21,63 @@ public partial class BreakInfoLine<T> : Container
 
         public Bindable<T> Current = new Bindable<T>();
 
-        private readonly OsuSpriteText text;
-        private readonly OsuSpriteText valueText;
+        private readonly LocalisableString name;
 
-        private readonly string prefix;
+        private OsuSpriteText valueText = null!;
 
-        public BreakInfoLine(LocalisableString name, string prefix = @"")
+        public BreakInfoLine(LocalisableString name)
         {
-            this.prefix = prefix;
+            this.name = name;
 
             AutoSizeAxes = Axes.Y;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours)
+        {
             Children = new Drawable[]
             {
-                text = new OsuSpriteText
+                new OsuSpriteText
                 {
                     Anchor = Anchor.Centre,
                     Origin = Anchor.CentreRight,
                     Text = name,
                     Font = OsuFont.GetFont(size: 17),
-                    Margin = new MarginPadding { Right = margin }
+                    Margin = new MarginPadding { Right = margin },
+                    Colour = colours.Yellow,
                 },
                 valueText = new OsuSpriteText
                 {
                     Anchor = Anchor.Centre,
                     Origin = Anchor.CentreLeft,
-                    Text = prefix + @"-",
+                    Text = @"-",
                     Font = OsuFont.GetFont(weight: FontWeight.Bold, size: 17),
-                    Margin = new MarginPadding { Left = margin }
+                    Margin = new MarginPadding { Left = margin },
+                    Colour = colours.YellowLight,
                 }
             };
-
-            Current.ValueChanged += currentValueChanged;
         }
 
-        private void currentValueChanged(ValueChangedEvent<T> e)
+        protected override void LoadComplete()
         {
-            string newText = prefix + Format(e.NewValue);
+            base.LoadComplete();
 
-            if (valueText.Text == newText)
-                return;
-
-            valueText.Text = newText;
+            Current.BindValueChanged(text => valueText.Text = Format(text.NewValue), true);
```

Comment:
Went too far here. The `BindValueChanged(..., true)` spec actually alters behaviour because it makes the initial `Text` spec of `"-"` on `valueText` useless (it will always be replaced by whatever `Format()` spits out at the point of `LoadComplete()`).